src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -97,8 +99,9 @@ const Comments = ({
   };
 
   return (
-    <div className={styles.container}>
+    <div className={classNames(styles.container)}>
```

Comment:
There's no point in `classNames` if there's only a single class. It's just a waste of CPU running the function.

Remove the import as well.